### PR TITLE
Add scope description of MLKube

### PR DESCRIPTION
**Pull request by ScorpioCPH**
@mitake @ddysher @DjangoPeng @gaocegege-caicloud @zjx-caicloud @YujiOshima PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable running machine learning infrastructure on top of Kubernetes.
+The primary focus of MLKube is:
+
+- Resource Management
+- Distributed Workloads Scheduling
```
Comment (gaocegege) [inline README.md]: I am a little confused, Do you mean ML workloads?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable running machine learning infrastructure on top of Kubernetes.
```
Comment (zjx-caicloud) [inline README.md]: A syntax error: s/enable running/ enable the running of Or enable us to run ?

File: README.md (hunk 1 above)
Comment (ScorpioCPH) [reply README.md]: Yes

Comment (gaocegege) [reply README.md]: Could we use ML instead of distributed since they have some differences :thinking:

File: README.md (hunk 2 above)
Comment (ScorpioCPH) [reply README.md]: done.

File: README.md (hunk 1 above)
Comment (ScorpioCPH) [reply README.md]: Update to `Distributed ML Workloads Scheduling`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+
+- Resource Management
+- Distributed ML Workloads Scheduling
+- Network
+- Storage
+
+## FAQ
+
+Q1: What is the different between this project and Kubeflow?
+A1: Kubeflow provides a high-level user-oriented abstraction/deployment of machine learning on Kubernetes.  This project focus on the low-level, not user-oriented, but infrastructure-oriented.
```
Comment (deyuanix) [inline README.md]: This is not a correct markdown syntax

Comment (ScorpioCPH) [reply README.md]: Done.

**Review by gaocegege — APPROVED** [review]

**Comment by gaocegege** [comment]
LGTM

File: README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable us to run machine learning infrastructure on top of Kubernetes.
```
Comment (DjangoPeng) [inline README.md:5]: s/infrastructure/workload, IMO.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
...
+The primary focus of MLKube is:
+
+- Resource Management
+- Distributed ML Workloads Scheduling
+- Network
```
Comment (DjangoPeng) [inline README.md:10]: We'd better explain the details of Network and Storage. I think we just focus on container network, am I right?

**Comment by mitake** [comment]
LGTM, thanks

File: README.md (hunk 4 above)
Comment (ScorpioCPH) [reply README.md:5]: I prefer `infrastructure` as this project is different from kubeflow :)

File: README.md (hunk 5 above)
Comment (ScorpioCPH) [reply README.md:10]: This is a good point, but i have no more details about this actually.

**Comment by ScorpioCPH** [comment]
@ddysher Ping.

File: README.md (hunk 5 above)
Comment (deyuanix) [reply README.md:10]: let's flesh this out later.

**Comment by deyuanix** [comment]
LGTM

File: README.md (hunk 4 above)
Comment (DjangoPeng) [reply README.md:5]: Got you! :)

**Pull request merged**